EIPS/eip-1.md

```diff
@@ -213,7 +215,7 @@ Images, diagrams and auxiliary files should be included in a subdirectory of the
 
 It occasionally becomes necessary to transfer ownership of EIPs to a new champion. In general, we'd like to retain the original author as a co-author of the transferred EIP, but that's really up to the original author. A good reason to transfer ownership is because the original author no longer has the time or interest in updating it or following through with the EIP process, or has fallen off the face of the 'net (i.e. is unreachable or isn't responding to email). A bad reason to transfer ownership is because you don't agree with the direction of the EIP. We try to build consensus around an EIP, but if that's not possible, you can always submit a competing EIP.
 
-If you are interested in assuming ownership of an EIP, send a message asking to take over, addressed to both the original author and the EIP editor. If the original author doesn't respond to email in a timely manner, the EIP editor will make a unilateral decision (it's not like such decisions can't be reversed :)).
+If you are interested in assuming ownership of an EIP, send a message asking to take over, addressed to both the original author and the EIP editor. If the original author doesn't respond to the email in time, the EIP editor will make a unilateral decision (it's not like such decisions can't be reversed :)).
```

Comment:
I believe `in a timely manner` is correct here.  There is no fixed time limit, and how long is acceptable is quite subjective.  "a timely manner" is a common way of expressing such a concept, while `in time` implies a hard deadline.

Reply:
I agree, `in a timely manner` is less formal and implies a softer deadline, albeit still seems a bit redundant. I do feel having a generous upper time limit will add clarity to the text as well as to the process. I am not sure there is precedent for such a proposed time limit though.

So in the meanwhile, I have reverted the text back to `in a timely manner`. I will keep this in mind as I get more familiar with the process and see if we need more details here.